REQUIRE

```diff
@@ -11,6 +11,7 @@ MacroTools
 StatsFuns 0.7.0
 SpecialFunctions
 Bijectors
+LogDensityProblems
```

Comment:
Since `LogDensityProblems` is only required by `DynamicHMC`, can we load it conditionally via `@require` too?